### PR TITLE
use cloudevent alias pkg

### PR DESCRIPTION
**Pull request by matzew**
as per title

/cc @n3wscott

### CODE REVIEW

File: pkg/kncloudevents/good_client.go [hunk 1]
```diff
@@ -1,26 +1,29 @@
 package kncloudevents
 
 import (
+	"github.com/cloudevents/sdk-go"
 	"github.com/cloudevents/sdk-go/pkg/cloudevents/client"
 	"github.com/cloudevents/sdk-go/pkg/cloudevents/transport/http"
 )
 
 func NewDefaultClient(target ...string) (client.Client, error) {
```
Comment (n3wscott) [inline pkg/kncloudevents/good_client.go]: This client too in the sig

File: pkg/kncloudevents/good_client.go [hunk 2]
```diff
@@ -1,26 +1,29 @@
 package kncloudevents
 
 import (
+	"github.com/cloudevents/sdk-go"
 	"github.com/cloudevents/sdk-go/pkg/cloudevents/client"
 	"github.com/cloudevents/sdk-go/pkg/cloudevents/transport/http"
```
Comment (n3wscott) [inline pkg/kncloudevents/good_client.go]: Delete these imports

File: pkg/kncloudevents/good_client.go [hunk 3]
```diff
@@ -1,26 +1,29 @@
...
-	t, err := http.New(tOpts...)
+	t, err := cloudevents.NewHTTPTransport(tOpts...)
 	if err != nil {
 		return nil, err
 	}
+
 	// Use the transport to make a new CloudEvents client.
-	c, err := client.New(t,
+	c, err := cloudevents.NewClient(t,
 		client.WithUUIDs(),
```
Comment (n3wscott) [inline pkg/kncloudevents/good_client.go]: This can come from cloudevents too

File: pkg/kncloudevents/good_client.go (hunk 2 above)
Comment (matzew) [reply pkg/kncloudevents/good_client.go]: @n3wscott  `Option` is still in `http`,

see:  https://github.com/cloudevents/sdk-go/pull/116

**Pull request merged**